packages/orbit-components/src/Table/__tests__/index.test.js

```diff
@@ -11,116 +12,109 @@ import TableCell from "../TableCell";
 import TableFooter from "../TableFooter";
 
 describe("Table", () => {
-  const compact = true;
-  const children = "Lorem ipsum dolor sit amet";
-  const dataTest = "test";
-  const type = "primary";
-  const striped = true;
-
-  const component = shallow(
-    <Table striped={striped} type={type} compact={compact} dataTest={dataTest}>
-      <TableHead>
-        <TableRow>
-          <TableCell>{children}</TableCell>
-          <TableCell>{children}</TableCell>
-          <TableCell>{children}</TableCell>
-          <TableCell>{children}</TableCell>
-        </TableRow>
-      </TableHead>
-      <TableBody>
-        <TableRow>
-          <TableCell>{children}</TableCell>
-          <TableCell>{children}</TableCell>
-          <TableCell>{children}</TableCell>
-          <TableCell>{children}</TableCell>
-        </TableRow>
-      </TableBody>
-    </Table>,
-  );
-
-  it("should have props", () => {
-    expect(component.find("Table__StyledTable").prop("compact")).toBe(compact);
-    expect(component.find("Table__StyledTable").prop("type")).toBe(type);
-    expect(component.find("Table__StyledTable").prop("striped")).toBe(striped);
+  it("should have data-test", () => {
+    const dataTest = "test";
+    render(<Table dataTest={dataTest}>kek</Table>);
+    expect(screen.getByTestId(dataTest)).toBeInTheDocument();
   });
 
-  it("should have rendered dataTest", () => {
-    expect(component.render().prop("data-test")).toBe(dataTest);
+  it("should have height 32px", () => {
```

Comment:
By not hardcoding the value we won't have to change the test name if the height changes.
```suggestion
  it("should have expected cell height", () => {
```